src/Views/Form/Templates/Sequoia/Sequoia.php

```diff
@@ -65,47 +65,54 @@ public function loadScripts() {
 		$templateOptions['payment_information']['header_label']   = ! empty( $templateOptions['payment_information']['header_label'] ) ? $templateOptions['payment_information']['header_label'] : __( 'Add Your Information', 'give' );
 		$templateOptions['payment_information']['checkout_label'] = ! empty( $templateOptions['payment_information']['checkout_label'] ) ? $templateOptions['payment_information']['checkout_label'] : __( 'Process Donation', 'give' );
 
-		wp_enqueue_style( 'give-google-font-montserrat', 'https://fonts.googleapis.com/css?family=Montserrat:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&display=swap', [], GIVE_VERSION );
-		wp_enqueue_style( 'give-sequoia-template-css', GIVE_PLUGIN_URL . 'assets/dist/css/give-sequoia-template.css', [ 'give-styles' ], GIVE_VERSION );
+		wp_enqueue_style( 'give-google-font-montserrat', 'https://fonts.googleapis.com/css?family=Montserrat:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&display=swap', array(), GIVE_VERSION );
+		wp_enqueue_style( 'give-sequoia-template-css', GIVE_PLUGIN_URL . 'assets/dist/css/give-sequoia-template.css', array( 'give-styles' ), GIVE_VERSION );
 
 		$primaryColor = $templateOptions['introduction']['primary_color'];
-		$dynamic_css  = sprintf(
-			'
+		$primaryColor = $templateOptions['introduction']['primary_color'];
+		$rawColor     = trim( $primaryColor, '#' );
+		$dynamic_css  = "
 			.seperator {
-				background: %1$s!important;
+				background: {$primaryColor}!important;
```

Comment:
Any reason for the lack of space between the value and the `!important`? I see later on in files we do have the space.

Reply:
Good catch, there shouldn't be a space in later files. I'll take a look at the other files affected in this PR.

Reply:
After reviewing code, it looks like our style pre-commit hooks add the space, so for consistency's sake I've added the space here as well: https://github.com/impress-org/givewp/pull/4713/commits/930bce02acd11918be6079100bc34c7250ddaad0